config.go

```diff
@@ -0,0 +1,85 @@
+package main
+
+// nolint: lll
+import (
+	"encoding/json"
+	"io/ioutil"
+
+	"github.com/brigadecore/brigade-cloudevents-gateway/internal/cloudevents"
+	"github.com/brigadecore/brigade-foundations/file"
+	"github.com/brigadecore/brigade-foundations/http"
+	"github.com/brigadecore/brigade-foundations/os"
+	"github.com/brigadecore/brigade/sdk/v2/restmachinery"
+	"github.com/pkg/errors"
+)
+
+// apiClientConfig populates the Brigade SDK's APIClientOptions from
+// environment variables.
+func apiClientConfig() (string, string, restmachinery.APIClientOptions, error) {
+	opts := restmachinery.APIClientOptions{}
+	address, err := os.GetRequiredEnvVar("API_ADDRESS")
+	if err != nil {
+		return address, "", opts, err
+	}
+	token, err := os.GetRequiredEnvVar("API_TOKEN")
+	if err != nil {
+		return address, token, opts, err
+	}
+	opts.AllowInsecureConnections, err =
+		os.GetBoolFromEnvVar("API_IGNORE_CERT_WARNINGS", false)
+	return address, token, opts, err
+}
+
+func tokenFilterConfig() (cloudevents.TokenFilterConfig, error) {
```

Comment:
Should we add a unit test for this function?

Reply:
Yes... oversight that it isn't. Will fix.